interface/src/app/sample/services.js

```diff
@@ -0,0 +1,37 @@
+angular.module('adage.sample.services', [
+  'ngResource',
+  'adage.utils'
+])
+
+.factory('Sample', ['$resource', '$http', 'ApiBasePath',
+  function($resource, $http, ApiBasePath) {
+    var Sample = $resource(
+      ApiBasePath + 'sample/:id/',
+      // TODO need to add logic for handling pagination of results.
+      // then, can change "limit" below to something sensible
+      {id: '@id', limit: 0},
```

Comment:
`@id` means `id` will be from input data (which is an argument for non_GET method). I don't think we need it here.